wiki/Chat_Console/th.md

```diff
@@ -1,102 +1,103 @@
-
-1. `Start Spectating`: ถ้าคุณและคนที่คุณส่องมีแมพเหมือนกัน คุณก็สามารถดูเขาเล่นได้ และชื่อของคุณจะไปปรากฎในรายชื่อผู้เฝ้าดู(?).
-2. `View Profile`: ดูโปรไฟล์ของผู้เล่นคนนั้น บนเบราว์เซอร์
-3. `Start Chat`: เปิดแทบแชทใหม่เพื่อคุยกับผู้เล่นคนนั้น
-4. `Invite to game`: (ถ้าคุณอยู่ในห้องมัลติ) ลากชวนเข้าห้องมัลติของคุณ
-5. `Add (Remove) as Friend`: เพิ่ม/เอา ผู้เล่นเข้าใน/ออกจาก รายชื่อเพื่อนของคุณ
-6. `Report User`: รายงานพฤติกรรมที่ไม่เหมาะสม ไม่ควรใช้นอกจากที่กล่าวไว้
-7. `Ignore User`: คำพูดใดๆ ที่ผู้เล่นพิมพ์ลงช่องแชทจะไม่ปรากฎในช่องแชทคุณ
-8. `Close`: ปิด Panel จบ
+| ![ผู้เล่นกำลังอยู่เฉย ๆ](img/Chat_Console-Idle.png "ผู้เล่นกำลังอยู่เฉย ๆ") | น้ำเงินเข้ม - ผู้เล่นไม่ได้ทำอะไร หรือคุยในแชทเฉยๆ |
+| ![ผู้เล่นกำลังเล่น](img/Chat_Console-Playing.png "ผู้เล่นกำลังเล่น") | เทา - กำลังเล่นคนเดียว |
+| ![ผู้เล่นกำลังรับชม](img/Chat_Console-Watching.png "ผู้เล่นกำลังรับชม") | ฟ้า - ดู รีเพลย์ หรือดูใครสักคนเล่นอยู่ |
+| ![ผู้เล่นกำลังแก้บีทแมพ](img/Chat_Console-Editing.png "ผู้เล่นกำลังแก้บีทแมพ") | แดง - แก้ไขบีทแมพตัวเอง |
+| ![ผู้เล่นกำลังทดสอบบีทแมพ](img/Chat_Console-Testing.png "ผู้เล่นกำลังทดสอบบีทแมพ") | ม่วง - ทดสอบบีทแมพ (ของตัวเองหรือคนอื่นก็ได้) |
+| ![ผู้เล่นกำลังอัพโหลดบีทแมพ](img/Chat_Console-Submitting.png "ผู้เล่นกำลังอัพโหลดบีทแมพ") | เขียวแกมน้ำเงิน - อัพโหลดบีทแมพ (ทั้งอัพโหลดใหม่หรืออัพเดท) ของตัวเองที่ทำไว้ |
+| ![ผู้เล่นกำลัง mod บีทแมพ](img/Chat_Console-Modding.png "ผู้เล่นกำลัง mod บีทแมพ") | เขียว - กำลัง Mod หรือแก้ไขบีทแมพของคนอื่น |
+| ![ผู้เล่นอยู่ในโหมดเล่นหลายคน](img/Chat_Console-Multiplayer.png "ผู้เล่นอยู่ในโหมดเล่นหลายคน") | น้ำตาล - อยู่ในโหลดเล่นหลายคน แต่ไม่ได้เล่นอยู่ |
+| ![ผู้เล่นกำลังเล่นในโหมดเล่นหลายคน](img/Chat_Console-Multiplaying.png "ผู้เล่นกำลังเล่นในโหมดเล่นหลายคน") | เหลือง - กำลังเล่นอยู่อยู่ในโหลดเล่นหลายคน |
+| ![ผู้เล่นไม่อยู่ (afk)](img/Chat_Console-Afk.png "ผู้เล่นไม่อยู่ (afk)") | ดำ - อยู่นิ่ง ๆ หรือไม่อยู่ที่คีย์บอร์ด (afk) |
+| ![ผู้เล่นกำลังเชื่อมต่อผ่าน IRC](img/Chat_Console-IRC.png "ผู้เล่นกำลังเชื่อมต่อผ่าน IRC") | น้ำเงินเข้ม (แต่ไม่มีรายละเอียดผู้เล่น) - ผู้เล่นที่ล็อกอินผ่านระบบ IRC หรือไม่สามารถดูสถิติการเล่นได้ |
+
+การคลิกที่ User Panel จะทำให้ตัวเลือกขึ้นมาบนหน้าจอ
+
+![ตัวเลือกบนหน้าจอ](img/TH-Chat_Console-Userpaneloptions.jpg "ตัวเลือกบนหน้าจอ")
+
+กดที่ปุ่มเลขหรือกดที่ปุ่มบนจอ เพื่อ:
+
+1. `เริ่มชมการเล่น`: ถ้าคุณและคนที่คุณส่องมีแมพเหมือนกัน คุณก็สามารถดูเขาเล่นได้ และชื่อของคุณจะไปปรากฏบนรายชื่อผู้เฝ้าดู
+2. `ดูโปรไฟล์`: ดูโปรไฟล์ของผู้เล่นคนนั้นบนเบราว์เซอร์
+3. `เริ่มพูดคุย`: เปิดแท็บแชทใหม่เพื่อคุยกับผู้เล่นคนนั้น
+4. `ชวนเข้าเล่นเกม`: (ถ้าคุณอยู่ในห้องโหมดเล่นหลายคน) ชวนเข้าห้องโหมดเล่นหลายคนของคุณ
+5. `เพิ่ม (เลิก) เป็นเพื่อน`: เพิ่มผู้เล่นเข้าหรือลบออกจากรายชื่อเพื่อนของคุณ
+6. `แจ้งรายงานผู้เล่นคนนี้...`: รายงานพฤติกรรมที่ไม่เหมาะสม ไม่ควรใช้นอกจากที่กล่าวไว้ คุณสามารถรายงานผู้ใช้ในเกมได้จากหลายอย่าง และบนเว็บ กับฟอรั่มเช่นกัน
+7. `เพิกเฉยพู้เล่น`: คำพูดใด ๆ ที่ผู้เล่นพิมพ์ลงช่องแชทจะไม่ปรากฏในช่องแชทคุณ
+8. `ปิด`: ปิด Panel
```

Comment:
```suggestion
8. `ปิด`: ปิด panel
```
Capitalization since in the previous article, Most English words are not capitalized